Assets/Scripts/SS3D/Systems/Inventory/Containers/Inventory.cs

```diff
@@ -112,7 +114,7 @@ protected override void HandleUpdate(in float deltaTime)
 
         public bool HasPermission(IDPermission permission)
         {
-            IIdentification id = IDContainer.Items.FirstOrDefault() as IIdentification;
+            IIdentification id = IDContainer.Container.Items.FirstOrDefault() as IIdentification;
```

Comment:
Couldn't you add a Items property to AttachedContainer that points to Container.Items? not sure if that is good practices but would reduce redundant code in my opinion

Reply:
done